networks/remote/ansible/roles/install/templates/systemd.service.j2

```diff
@@ -8,7 +8,7 @@ Restart=on-failure
 User={{service}}
 Group={{service}}
 PermissionsStartOnly=true
-ExecStart=/usr/bin/tendermint node --proxy_app=dummy
+ExecStart=/usr/bin/tendermint node --proxy_app=persistent_kvstore --p2p.persistent_peers=167b80242c300bf0ccfb3ced3dec60dc2a81776e@165.227.41.206:46656,3c7a5920811550c04bf7a0b2f1e02ab52317b5e6@165.227.43.146:46656,303a1a4312c30525c99ba66522dd81cca56a361a@159.89.115.32:46656,b686c2a7f4b1b46dca96af3a0f31a6a7beae0be4@159.89.119.125:46656
```

Comment:
where did you get these IPs from? we don't know IPs at this point. /cc @greg-szabo

Reply:
they're what terraform spun up. Agree that they shouldn't be hardcoded - this is placeholder.